tests/tests/test_contacts/test_mixing.py

```diff
@@ -0,0 +1,25 @@
+import phoebe
+import numpy as np
+
+
+def test_mixing(verbose=False):
+    if verbose:
+        print('phoebe.default_contact_binary()')
+    b = phoebe.default_contact_binary()
+    b.add_dataset('mesh', compute_times=[0])
+    b.set_value('teff@secondary', 3000)  # so we have different temperatures
+    b.set_value('mixing_enabled', True)
+    b.set_value('mixing_method', 'perfect')  # since other mixing methods are arbitrarily parametrized, test this case only
+    b.set_value('columns', '*')  # to populate 'teffs'
+
+    assert b.run_checks().passed
+
+    b.run_compute()
+
+    assert (res := abs(1 - np.mean(b['teffs@secondary'].value) / np.mean(b['teffs@primary'].value))) <= 0.01
+    if verbose:
+        print('relative teff difference', res)
```

Comment:
same here

Reply:
none of the tests use loggers. I only find the `if verbose: print()` construction

Reply:
fair enough, but I would want to move to loggers in the future, and I'll be using them in all tests that I write because they provide granularity that print() does not.